packages/eos-components/src/components/CardBody.vue

```diff
@@ -32,8 +32,7 @@
 
 <script>
 import VClamp from 'vue-clamp';
-import { mapGetters } from 'vuex';
-import { StructuredTags } from '@/constants';
+import { StructuredTags, getFirstStructuredTag } from './utils';
```

Comment:
StructuredTags is a constant so it should go to the new `eos-components/src/constants.js`.